### DISCUSSION TITLE
Welcome to astro-reactive-library Discussions!

### DISCUSSION THREAD

**Discussion by ayo-run** [ayoayco/astro-reactive-library#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jayvi** [comment]
when submit the form how to define url and how you sending the values from the form

**Reply by ayo-run** [reply]
we started discussing the implementation in this issue: https://github.com/ayoayco/astro-reactive-library/issues/236

**Reply by ayo-run** [reply]
Hello @jayvi we just released a new version for the Form package with two new properties: `action` and `method`

Setting the action property value as the URL where you want the values to be handled

The `method` prop could be set to 'POST', 'GET', etc.

**Reply by jayvi** [reply]
super cant wait to test it

**Reply by ayo-run** [reply]
@jayvi Let us know what you find or if you have a use case that is not supported. Maybe we can help. We also have a Discord if you want to join :)

**Reply by ayo-run** [reply]
If anyone needs here is the updated docs for the Form component properties: https://docs.astro-reactive.dev/en/api/form/form-component/#properties
